### ISSUE TITLE
bounding box's form

### ISSUE THREAD

**Issue by YKritet** [AlexeyAB/darknet#3444]
I would like to change the bounding box's form since the object I want to detect is so small that it gets covered by the label which dose't allow me to judge the accuracy of the detector. Could you please show me the snippet of code that deals with this.

**Comment by hitle451997** [comment]
you mean anchor boxes ? refer : 
darknet.exe detector calc_anchors data/obj.data -num_of_clusters 9 -width 832 -height 832

**Comment by YKritet** [comment]
Yes, for every object detected, a text ( label or name of class depending on terminology ) appears on top of the box that surrounds it. I want to know how to change the size of that text. 
I tested the solution you provided, but it only gave me the anchor boxes sides' coordinates.

**Comment by danzigjabalpur** [comment]
in image.c L415, you can comment it out to hide the label
[`draw_label(im, top + width, left, label, rgb);`](https://github.com/AlexeyAB/darknet/blob/cc41339805d8b48edef67d0dcf4925b45c212277/src/image.c#L415)

what I want to know is how to change the bbox colours

**Comment by YKritet** [comment]
Thanks a lot, for box color checkout darknet.py. 
```
 boxColor = (int(255 * (1 - (confidence ** 2))), int(255 * (confidence ** 2)), 0)
                draw.set_color(image, (rr, cc), boxColor, alpha= 0.8)
                draw.set_color(image, (rr2, cc2), boxColor, alpha= 0.8)
                draw.set_color(image, (rr3, cc3), boxColor, alpha= 0.8)
                draw.set_color(image, (rr4, cc4), boxColor, alpha= 0.8)
                draw.set_color(image, (rr5, cc5), boxColor, alpha= 0.8)
```
